ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -216,6 +216,15 @@
 "REACTION_TIME_TASK_ATTEMPTS_FORMAT" = "Attempt %@ of %@";
 "REACTION_TIME_TASK_ACTIVE_STEP_TITLE" = "Quickly shake the device when the blue circle appears";
 
+/* Tower of Hanoi active task */
+"TOWER_OF_HANOI_TASK_TITLE" = "Tower of Hanoi";
+"TOWER_OF_HANOI_TASK_INTENDED_USE" = "This activity evaluates your puzzle solving skills.";
+"TOWER_OF_HANOI_TASK_TASK_INTRO_TEXT_FORMAT" = "Move the entire stack to the highlighted platform in as few moves as possible.";
+"TOWER_OF_HANOI_TASK_TASK_CALL_TO_ACTION" = "Tap Get Started to begin";
+"TOWER_OF_HANOI_TASK_ACTIVE_STEP_INTRO_TEXT" = "Solve the Puzzle \n";
```

Comment:
The problem with that approach is that the actual number of lines can be higher than 2 depending on localization and screen width. We solved a similar problem on the *Memory Game Task* by defining `ORKScreenMetricMinimumStepHeaderHeightForMemoryGame`. Have a look [here](https://github.com/ResearchKit/ResearchKit/blob/master/ResearchKit/ActiveTasks/ORKSpatialSpanMemoryStepViewController.m#L157).

If I remember properly, that metric guarantees that the layout will be consistent if the title is 3 or less lines (all of Memory Game's localized titles seem to be 3 lines or less on all devices).